code/game/mecha/mecha.dm

```diff
@@ -1171,7 +1171,7 @@
 		if(istype(mmi_as_oc, /obj/item/mmi/robotic_brain))
 			var/obj/item/mmi/robotic_brain/R = mmi_as_oc
 			if(R.imprinted_master)
-				to_chat(brainmob, "<span class='notice'>Your imprint to [R.imprinted_master] has been temporarily disabled. You should help the crew and not commit harm.</span>")
+				to_chat(brainmob, "<span class='warning'><FONT size = 3><B>Your imprint to [R.imprinted_master] has been temporarily disabled. You should help the crew and not commit harm.</B></FONT></span>")
```

Comment:
try using the span class `biggerdanger` or `userdanger` and see how that looks.

Reply:
![biggerdanger](https://user-images.githubusercontent.com/12178527/163495928-70ffd28d-373b-4b92-b9fa-4723193f13af.png)

Right. I tried both. They look basically the same as each other, however I think  `biggerdanger` does look better than warning. I'll be swapping it to that.

Reply:
sorry I don't think I was clear enough, I did mean this instead of manually editing the font size. `biggerdanger` is defined as `.biggerdanger				{color: #ff0000;	font-weight: bold; font-size: 150%;}`

Reply:
Fixed it.